source/docs/appendices/migration-guides.md

```diff
@@ -9,7 +9,47 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.14.0` -> `2.15.0`
 
-Nothing yet!
+### Node Version Support Change
+
+We now support node versions 14, 16, 17. So if you are using node 12 you will need to update your node version to one supported (16 recommended).
+
+In order to support node 17 we made changes that might impact your codebase:
+
+- `@apollo/*` use is prohibited. `@apollo/client` is only installed in Front-Commerce because it is needed by one of our dependencies. Other `@apollo/*` dependencies are not a `front-commerce` direct dependency and therefore should not be used. To use `2.15.0` you need to remove all uses of `@apollo/*` usages in your codebase and replace them with `react-apollo`, or `graphql-tag` imports where applicable. Check [here](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/1215/diffs?commit_id=8fae8438d46200cbd310b16c30b3078da28b7d42) and [here](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/1215/diffs?commit_id=418a291f917660b18672b6be2eb2778cb51dfef9)/ e.g:
+
+```diff
+-import { useLazyQuery, useQuery, useMutation } from "@apollo/react-hooks";
++import { useLazyQuery, useQuery, useMutation } from "react-apollo";
+```
+
+- resolvers for fields not in the `schema` now throws errors. To avoid this remove all unneeded `field`/`Type` resolvers that do not have a representation in the GraphQL `schema`. Check our [cleaning commit](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/1216/diffs?commit_id=1e2e04e7f28dba2ebd36da50701478a13a58a238) regarding this issue.
```

Comment:
```suggestion
- Resolvers for fields not in the `schema` now throw errors. To avoid this remove all unneeded `field`/`Type` resolvers that do not have a representation in the GraphQL `schema`. Check our [cleaning commit](https://gitlab.com/front-commerce/front-commerce/-/merge_requests/1216/diffs?commit_id=1e2e04e7f28dba2ebd36da50701478a13a58a238) regarding this issue.
```